.travis.yml

```diff
@@ -1,94 +1,69 @@
-      script:
-        - yarn cypress install
-        - yarn db:reseed
-        - cd backend/core
-        - yarn nest start &
-        - cd ../../sites/public
-        - yarn build
-        - yarn start -p 3000 &
-        - yarn wait-on "http-get://localhost:3000" && yarn cypress run
-        - kill $(jobs -p) || true
-
+  - script: yarn test:shared:ui
+    name: UI components tests
+  - script: yarn build:app:public
+    name: Build public site
+  - script: yarn build:app:partners
+    name: Build partners site
+  - script: yarn test:backend:core:testdbsetup && yarn test:backend:core
+    name: Backend unit tests
+  - script: yarn test:e2e:backend:core
+    name: Backend e2e tests
+  - script: yarn test:public:unit-tests
+    name: Public site unit tests
+  - stage: longer tests
+    name: Partners site Cypress tests
+    script:
+    - yarn cypress install
+    - cd backend/core
+    - yarn db:reseed:detroit
+    - yarn nest start &
+    - cd ../../sites/partners
+    - yarn build
+    - yarn start -p 3001 &
+    - yarn wait-on "http-get://localhost:3001" && yarn cypress run
+    - kill $(jobs -p) || true
+  - stage: longer tests
+    name: Public site Cypress tests
+    script:
+    - yarn cypress install
+    - yarn db:reseed
+    - cd backend/core
+    - yarn nest start &
+    - cd ../../sites/public
+    - yarn build
+    - yarn start -p 3000 &
+    - yarn wait-on "http-get://localhost:3000" && yarn cypress run
+    - kill $(jobs -p) || true
 addons:
-  postgresql: "11"
+  postgresql: '11'
   apt:
     packages:
-      - postgresql-11
-      - postgresql-client-11
-      # if using Ubuntu 16 need this library
-      # # https://github.com/cypress-io/cypress-documentation/pull/1647
-      - libgconf-2-4
-
+    - postgresql-11
+    - postgresql-client-11
+    - libgconf-2-4
 env:
-  global: PGPORT=5433
-    PGUSER=travis
-    TEST_DATABASE_URL=postgres://localhost:5433/bloom_test
-    REDIS_TLS_URL=redis://127.0.0.1:6379/0
-    NEW_RELIC_ENABLED=false
-    NEW_RELIC_LOG_ENABLED=false
+  global:
+  - PGPORT=5433 PGUSER=travis DATABASE_URL=postgres://localhost:5433/bloom_detroit
+    TEST_DATABASE_URL=postgres://localhost:5433/bloom_test REDIS_TLS_URL=redis://127.0.0.1:6379/0
+    NEW_RELIC_ENABLED=false NEW_RELIC_LOG_ENABLED=false
+  - secure: iVBAxWMlvmBTcSq5qQhASJauYaL9IhA45e4J2WCL9ybf07aAqbpTSPG7O/V+ZBQ7mHRx9Io6U5SZVIYOhKoCwjIdSQeyzikeppRFJZwgC1T1cPqeIIWLRNjSVL+kmyPSjwZxNZTJsferlQYX/wLHteX3Rq6/kBvb2WKjwNu/mF66y7xdER0JpLgMQ3V6bGz1Ji16GVY7LYsbOdwCI1pGH/Q3swCXnLZlBixRtCEy+yFQzaz60WZgjLlDyM60+5TcvtcZFPZuO5LNyJd29iSqbR2t0EsvBTxs9j9GC8a4ySLcPK9qMOme2q3O8b/5CzC1CmjUW6XmXsd5LXyHNV+JTx/DotkTzk5k972z8awxzG+VDiTEO04zGqUQfhRi2IAgGYz0zFsRwO5pKLEnbQz5LOCC12zy1s+2d2OE9OoW/xOEQjVz/4QJr8Z49UWze1p5Y3rRUx4+++Di0A8vPufYHAkmUmxZP/Qguj0ngAn+GnFk/82UpilW/HWwkDUit6pz+xVvDTJ9eCkJYwfZkrAw3qFYm73PkYkI1HR0puMKX0V/rGY3r18nZzPpzUltAZQblSZKexqpcZKGu8avcNxHUXtKS6BY462XxCG4HUuwyrRH9N7e5Tos13rI4X18jCnWdKQ0jB4szpFOfqxqzuvyQ7UMwRYt0qantXhkDVo7qvo=
```

Comment:
Can we remove these?

Reply:
@YazeedLoonat , once you address this, we're good to merge. I also create a follow-up issue per @emilyjablonski comment. https://github.com/CityOfDetroit/bloom/issues/1308